src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const threeDaysDiscount = 20;
+  const weekDiscount = 50;
+  const allDays = days * rentPrice;
+
+  return days >= 7 ? allDays - weekDiscount : days >= 3
+    ? allDays - threeDaysDiscount : allDays;
```

Comment:
Nested ternaries is a really bad practice. It'd be better to use simple `if`'s if you have multiple conditions